.gitattributes

```diff
@@ -0,0 +1,23 @@
+* text=lf
+
+*.txt text
+*.asp text
+*.aspx text
+*.c text
+*.jsp text
+*.java text
+*.xml text
+*.php text
+*.pl text
+*.cgi text
+*.cfm text
+*.gradle text
+*.gradle.kts text
+*.properties text
+*.md text
+
+*.bat text eol=crlf
+*.sh=text
```

Comment:
I believe this should be a space not an equals sign.

Reply:
Oh sorry, my mistake